spec/requests/admin_requests_spec.rb

```diff
@@ -9,22 +9,51 @@
       expect(response).to be_successful
     end
 
-    context "with rendered views" do
+    it "shows a logout button" do
+      get admin_dashboard_path
+      expect(response.body).to match(/log out/im)
+    end
+
+    context "when the user has a name" do
       let!(:users_list) { create_list(:user, 25) }
-      let!(:user) { create(:user, name: "Name Not Provided") }
-      edit_user_path_regex = Regexp.new('admin/users/[0-9]+/edit\\?organization_id=admin')
+      let!(:user_with_name) { create(:user, name: "John Doe", email: "john@example.com") }
+
+      it "displays the users list" do
```

Comment:
Let's go ahead and drop this block (and `users_list`) -- I don't think we need it compared to the rest of the name/no-name examples.